### PR TITLE
Fix some typos in java doc

### PR DESCRIPTION
**Pull request by bmscomp**
This is a too minor pull request that is aims to fix some typos in java doc for `CucumberOptions` class

### CODE REVIEW

File: runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
     String[] features() default {};
 
     /**
-     * Package to load glue code (step definitions, hooks and plugins) from.
+     * Package to load glue code (step definitions, hooks, and plugins) from.
```
Comment (gastaldi) [inline runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java:43]: I think this is correct

File: runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
     String[] features() default {};
 
     /**
-     * Package to load glue code (step definitions, hooks and plugins) from.
```
Comment (gastaldi) [inline runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java:43]: I believe this is correct

File: runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java [hunk 3]
```diff
@@ -54,7 +54,7 @@
     String[] glue() default {};
 
     /**
-     * Package to load additional glue code (step definitions, hooks and
```
Comment (gastaldi) [inline runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java:57]: This is correct also

File: runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java (hunk 2 above)
Comment (bmscomp) [reply runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java:43]: Yes right, I figured it out after some revisions of my english courses. It is a called `serial comma` or` Oxford comma `

File: runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java (hunk 3 above)
Comment (bmscomp) [reply runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java:57]: Yes I agree both of them are correct

**Comment by bmscomp** [comment]
@gastaldi  The pull request is updated, for fit review comments

**Comment by gastaldi** [comment]
Thanks!

**Pull request merged**